assets/sass/config/_variables.scss

```
@@ -223,6 +225,10 @@ $c-entity-search-autocomplete-bg-hover: rgba($c-silver, 0.2);
 $c-setup-analytics-notice-bg: $c-alabaster;
 $c-setup-analytics-notice-text: $c-cape-cod;
 
+// Analtyics CTAs
```

@techanvil Small typo here 😁 
```suggestion
// Analytics CTAs
```

---

Thanks for the spot!